<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>200</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>525</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tool_tab">
        <attribute name="title">
         <string>Instances</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Class</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="class_combo"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="name_edit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="instances_list"/>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>194</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="view_tab">
        <attribute name="title">
         <string>View</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Show</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="show_initial_points">
              <property name="text">
               <string>Initial points</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_cameras">
              <property name="text">
               <string>Cameras</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_cropped">
              <property name="text">
               <string>Cropped</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_bounding_boxes">
              <property name="text">
               <string>Boxes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_filtered_points">
              <property name="text">
               <string>Filter labels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_color_instances">
              <property name="text">
               <string>Color instances</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>View mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="view_mode"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Debug</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QCheckBox" name="fixed_size">
              <property name="text">
               <string>Fixed size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fixed_opacity">
              <property name="text">
               <string>Fixed opacity</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>165</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="camera_tab">
        <attribute name="title">
         <string>Camera</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Move Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="move_speed">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Animate Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="animate_speed">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Zoom rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="zoom_rate">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>306</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInstances"/>
   <addaction name="actionEdit_Foreground"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
  </widget>
  <action name="actionInstances">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="mdi-brush">
     <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Annotate Instances</string>
   </property>
  </action>
  <action name="actionEdit_Foreground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit Foreground</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
